backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -54,6 +54,20 @@ def _ad_hoc_detect_and_delete_discarded_drafts(schema: str = SCHEMA, local: bool
         handle_discarded_drafts(con, csets_and_members, set(draft_ids), report_success=False, use_local_db=local)
 
 
+def _ad_hoc_sync_draft_expression_items(since='2023-08-15', schema=SCHEMA, local=False):
```

Comment:
Completed successfully, syncing all expression items since this date, which is when we added the drafts feature.